draft-sasikumar-id-claim-169.md

```diff
@@ -70,57 +70,228 @@ Claim 169 represents a JSON Object that includes the below table as ID attribute
+
+| `62`      | `[Biometrics]` | Face                 | Person's Face biometrics                                                              |
+
+| `63`      | `[Biometrics]` | Right Palm Print     | Person's Right Palm Print biometrics                                                   |
+
+| `64`      | `[Biometrics]` | Left Palm Print      | Person's Left Palm Print biometrics                                                    |
+
+| `65`      | `[Biometrics]` | Voice Biometrics      | Person's Voice biometrics                                                              |
+
+| `66.. 74` |                | Reserved             | Reserved for future Person's Biometrics Data attributes                                 |
+
+| `75.. 99` |                | Reserved             | Reserved for future attributes                                                         |
+
+### `Biometrics`
+
+| Attribute | Type   | Attribute Name                       | Description                                                                                                                                                                                          |
+|-----------|--------|--------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `0`       | `bstr` | Data                                 | Biometrics binary data                                                                                                                                                                               |
+| `1`       | `int`  | [Data format](#data-formats)         | Optional biometrics data format.                                                                                                                                                                     |
+| `2`       | `int`  | [Data sub format](#data-sub-formats) | Optional biometrics data sub format.                                                                                                                                                                 |
+| `3`       | `tstr` | Data issuer                          | Optional biometric data issuer.                                                                                                                                                                      |
+
+#### Data formats
+
+| Data format | Description |
+|-------------|-------------|
+| `0`         | Image       |
+| `1`         | Template    |
+| `2`         | Sound       |
+| `3`         | Bio hash    |
+
+##### Data sub formats
+
+###### Image
+
+| Subformat | Description |
+|-----------|-------------|
+| `0`       | PNG         |
+| `1`       | JPEG        |
+| `2`       | JPEG2000    |
+| `3`       | AVIF        |
+| `4`       | WEBP        |
+| `5`       | TIFF        |
+| `6`       | WSQ         |
```

Comment:
```suggestion
| `6`       | WSQ         |
| `100..200` | Vendor specific                  |
```
Here will be useful to also have a vendor specific region, we already have that use case.